sdk/python/kfp/deprecated/cli/cli.py

```diff
@@ -32,6 +32,7 @@
     'components'
 ]
 
+
```

Comment:
Just one more line in the diff :)

Consider using `git checkout upstream/master sdk/python/kfp/deprecated/cli/cli.py` (assuming the kubeflow/pipelines remote is named `upstream` in your local version of the repo... `origin/master` should also work)